user/languages/python.md

```diff
@@ -199,6 +198,21 @@ For a real world example, see [getsentry/sentry](https://github.com/getsentry/se
 For Python projects, `env` and `python` can be given as arrays
 to construct a build matrix.
 
+```yaml
```

Comment:
Not sure if this example adds much value here. There are examples using `python`, and `allow_failures` is considered elsewhere.

Reply:
The reason that I added allow_failures was because this is a year of porting in the Python world with the upcoming EOL of Python 2, the performance improvements of Python 3, and the Django 1 to 2 transition.  More people could be using CI to smoothly navigate these transitions if they understood the allow_failures approach so we should not hide it.

Let me know if you need it removed.

Reply:
Hey @cclauss - if you could remove the example I'll merge this in.

> More people could be using CI to smoothly navigate these transitions if they understood the allow_failures approach so we should not hide it.

This might make more sense as a separate, language independent page on how to manage to version number transition on Travis CI using `allow_failure`, what do you think?